apps/site/src/components/pages/dashboard/dashboard-wordpress-section.tsx

```diff
@@ -0,0 +1,213 @@
+import { faClipboard, faEye } from "@fortawesome/free-regular-svg-icons";
+import { Box, Stack, Typography, typographyClasses } from "@mui/material";
+import { useRef, useState } from "react";
+
+import { apiClient } from "../../../lib/api-client";
+import { CODE_FONT_FAMILY } from "../../../theme/typography";
+import { Button } from "../../button";
+import { FontAwesomeIcon } from "../../icons";
+import { Link } from "../../link";
+
+const defaultApiKey =
+  "b10ck5.9faa5da6664f7229999439d5433d4ac2.c549e923-c3b6-451f-baba-20b391e6033d";
+
+export const DashboardWordpressSection = ({
+  wordpressInstanceUrl,
+}: {
+  wordpressInstanceUrl: string;
```

Comment:
I suspect that we are going to store both
(a) the WP instance URL (e.g. https://example.com) and
(b) the WP BP plugin settings page to send them back to (https://example.com/?????)

One potential approach would be to send (b) when signing up, store the origin as `wordpressInstanceUrl` as we do now in the db, and store the full settings page URL in the db (but only need one value for it).

I don't think we can rely on the path to the settings page being the same across instances because people can change the admin path (defaults to `/wp-admin`).

I think you could look at this as part of doing the WP side of it though, as it'll be easier to check / iterate.

Reply:
I've done it as part of this PR.